src/main/java/com/bergerkiller/bukkit/tc/signactions/SignActionTicket.java

```diff
@@ -1,30 +1,37 @@
 package com.bergerkiller.bukkit.tc.signactions;
 
+import com.bergerkiller.bukkit.common.utils.ParseUtil;
+import com.bergerkiller.bukkit.tc.Localization;
 import com.bergerkiller.bukkit.tc.Permission;
-import com.bergerkiller.bukkit.tc.TCConfig;
+import com.bergerkiller.bukkit.tc.TrainCarts;
+import com.bergerkiller.bukkit.tc.controller.MinecartMember;
 import com.bergerkiller.bukkit.tc.events.SignActionEvent;
 import com.bergerkiller.bukkit.tc.events.SignChangeActionEvent;
-/*
-import com.earth2me.essentials.register.payment.Method;
-import com.earth2me.essentials.register.payment.Method.MethodAccount;
-import com.earth2me.essentials.register.payment.Methods;
-*/
+import com.bergerkiller.bukkit.tc.properties.TrainProperties;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Player;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
 
 /**
  * @author reeZZer
- *         Me (Bergerkiller) gives a big thank-you to him for writing the economics for this feature :)
+ * Me (Bergerkiller) gives a big thank-you to him for writing the economics for this feature :)
  */
 public class SignActionTicket extends SignAction {
 
     @Override
     public boolean match(SignActionEvent info) {
-        return TCConfig.EssentialsEnabled && info.isType("ticket");
+        return info.isType("ticket");
```

Comment:
Did you also remove the config option inside TCConfig for it?

Reply:
Its is removed since commit 75ea6594